Doc/whatsnew/3.12.rst

```diff
@@ -1571,8 +1571,8 @@ zipimport
 Others
 ------
 
-* Removed the ``suspicious`` rule from the documentation Makefile, and
-  removed ``Doc/tools/rstlint.py``, both in favor of `sphinx-lint
+* Remove the ``suspicious`` rule from the documentation Makefile and
+  :file:`Doc/tools/rstlint.py`, both in favor of `sphinx-lint
```

Comment:
Note that this change to using ``:file:`` is included to avoid merge conflicts due to potential out-of-order merges